### PR TITLE
fix(Slider): Fixed vertical & horizontal slider icons misalignment

### LINKED ISSUE #753: Vertical slider icons are misaligned
Might need to do some cleanup and fixes on how things are spaced/aligned in the vertical slider.

_Originally posted by @veekeys in https://github.com/iTwin/iTwinUI/pull/718#pullrequestreview-1086899678_

### PR DESCRIPTION
**Pull request by rohan-kadkol**
Closes #753 
Moved from PR #754 

- Removed the hardcoded `width` in `slider.html` for the min and max labels of the horizontal and vertical sliders. Replaced it with styling in the `slider.scss`. [(as mentioned here)](https://github.com/iTwin/iTwinUI/issues/753#issuecomment-1231652732)
- Fixed vertical slider's max and min labels misallginment as mentioned in #753 
- Also, improved the alignment of the min and the max labels in the horizontal slider (as they were similar to what was done to fix the vertical slider)

### TODO
- [x] Change base of PR to `v1` branch

### CODE REVIEW

**Review by gretanausedaite — APPROVED** [review]
I have no comments. Looks good.

File: packages/itwinui-css/src/slider/slider.scss [hunk 1]
```diff
@@ -38,6 +40,8 @@ $tick-height: var(--iui-size-s); // 12px
 
 @mixin iui-slider-rail {
   position: absolute;
+  width: 100%;
+  height: $track-height;
```
Comment (elephantcatdog) [inline packages/itwinui-css/src/slider/slider.scss:44]: It doesn't seem to me like the height and width need to be defined here if they're being defined under `@mixin iui-slider-horizontal > .iui-slider-rail` and `@mixin iui-slider-vertical > .iui-slider-rail`. Why did you add these here?

**Review by elephantcatdog — COMMENTED** [review]
It looks so much better!

File: packages/itwinui-css/src/slider/slider.scss (hunk 1 above)
Comment (rohan-kadkol) [reply packages/itwinui-css/src/slider/slider.scss]: **[DONE]** Ohh, right, thanks! These might have accedentally stayed in, but are not needed. Removed it now.

**Comment by rohan-kadkol** [comment]
I'm surprised why all the tests are not showing up or running for this PR. It currently says `1 successful check`. But it is not running the visual tests. Is this expected?

**Comment by bentleyvk** [comment]
> I'm surprised why all the tests are not showing up or running for this PR. It currently says `1 successful check`. But it is not running the visual tests. Is this expected?

Because you are merging to `v1` branch, no the `main`.

**Review by elephantcatdog — APPROVED** [review]
Looks good to me!
